### PR TITLE
Use gettext-setup gem for internationalization

### PR DESCRIPTION
**Pull request by smcclellan**
In order for error messages generated by this client to be translated, we
need to pass the error messages through gettext. This pulls in the
gettext-setup gem and passes error messages through the gettext method for
translation.

Fixes https://tickets.puppetlabs.com/browse/RAZOR-858

### CODE REVIEW

File: locales/config.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  # This is used in comments in the .pot and .po files to indicate what
+  # project the files belong to and should be a little more descriptive than
+  # <project_name>
+  package_name: Razor Client
+  # The locale that the default messages in the .pot file are in
+  default_locale: en
+  # The email used for sending bug reports.
+  bugs_address: docs@puppet.com
+  # The holder of the copyright.
+  copyright_holder: Puppet, LLC.
```
Comment (steveax) [inline locales/config.yaml]: Puppet Inc.?

File: spec/locales/config.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  # This is used in comments in the .pot and .po files to indicate what
+  # project the files belong to and should be a little more descriptive than
+  # <project_name>
+  package_name: Razor Client
+  # The locale that the default messages in the .pot file are in
+  default_locale: en
+  # The email used for sending bug reports.
+  bugs_address: docs@puppet.com
+  # The holder of the copyright.
+  copyright_holder: Puppet, LLC.
```
Comment (steveax) [inline spec/locales/config.yaml]: Puppet Inc.?

File: locales/config.yaml (hunk 1 above)
Comment (smcclellan) [inline locales/config.yaml]: I'll update this. It was copied over from what Razor had on the server-side.

**Comment by steveax** [comment]
LGTM except for the Puppet LLC - Puppet Inc. changes.

**Comment by smcclellan** [comment]
Fixed the LLC issues.

**Comment by steveax** [comment]
👍

**Pull request merged**